charts/karmada/values.yaml

```diff
@@ -28,22 +28,42 @@ components: []
   #   "search"
   # ]
 
+cfssl:
+  ## @param image.registry karmada cfssl image registry
+  ## @param image.repository karmada cfssl image repository
+  ## @param image.tag karmada cfssl image tag (immutable tags are recommended)
+  ##
+  image:
+    registry: docker.io
+    repository: cfssl/cfssl
+    tag: latest
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ##
+    pullPolicy: IfNotPresent
+
+kubectl:
+  ## @param image.registry karmada kubectl image registry
+  ## @param image.repository karmada kubectl image repository
+  ## @param image.tag karmada kubectl image tag (immutable tags are recommended)
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/kubectl
+    tag: latest
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ##
+    pullPolicy: IfNotPresent
+
 ## pre-install job config
 preInstallJob:
```

Comment:
PreInstallJob、postInstallJob、postDeleteJob should be deleted.

Reply:
I did not remove these three extra configurations, because they may have other parameters to be set in addition to the images repo